test/components/meta_test.dart

```diff
@@ -57,6 +67,29 @@ List<GeoJSONObject> featureAndCollection(GeometryObject geometry) {
 }
 
 main() {
+  test('propEach', () {
+    collection(pt).forEach((input) {
+      propEach(input, (prop, i) {
+        expect(prop, {'a': 1});
+        expect(i, 0);
+      });
+    });
+  });
+
+  test('featureEach', () {
+    collection(pt).forEach((input) {
+      featureEach(input, (feature, i) {
+        expect(feature.properties, {'a': 1});
+        expect(
+            feature.geometry,
+            Point.fromJson({
+              'coordinates': [0, 0],
+            }));
+        expect(i, 0);
+      });
+    });
+  });
+
```

Comment:
so, it looks like it may be difficult to implement the same `meta -- breaking of iterations` test against multiple callback types with dart, but we are missing a test for featureEach similar to https://github.com/Turfjs/turf/blob/master/packages/turf-meta/test.js#L1342

for example line 205 of this file should have added `featureEach` to the list of functions, but I'm not sure how tohandle the strong callback typing in dart. Perhaps that test needs to be refactored into some helper functions so we can properly duplicate that test coverage.

Reply:
Implemented two `--breaking of iterations` tests to cover the use-case. Thanks for flagging!